Module 2 - How do I connect to the cluster.md

```diff
@@ -14,9 +14,49 @@ By the end of this module, you will be able to:
 
 # Connecting to the Cluster using SSH
 
-- What is SSH?
+<!-- - What is SSH?
 - How to connect to the cluster using SSH?
-- How to set up SSH keys?
+- How to set up SSH keys? -->
+
+## What is SSH?
+
+SSH or Secure Shell is a network protocol and a software that lets you control a computer remotely over the internet. It uses encryption to establish a secure connection between your local machine and a remote server, like an HPC cluster. You can access the cluster through your computer's command line using SSH to run jobs and transfer files to and from the cluster.
+
+
+## How to connect to the cluster using SSH?
+
+To connect to the cluster using SSH, you need a local machine (your laptop) with an SSH client software installed and internet access. Most operating systems come pre-installed with an SSH client. You can check if your computer has an SSH client by running the following command in your terminal:
+
+```bash
+ssh -V
+```
+
+If you see the version information of the SSH client, you have it installed and are ready to connect to the cluster. Next, you need the address or hostname of the cluster. The hostname tells the SSH client where to find the cluster. You also need a username on the cluster to log in. Once you have the hostname and username, you can connect to the cluster using the following command:
+
+```bash
+ssh usename@hostname
+```
+
+For example, to connect to the cluster with the hostname `supercomputer.awesome.ca` and the username `andy`, you would run,
+
+```bash
+ssh andy@supercomputer.awesome.ca
+```
+
+When you run this command, you are essentially trying to log in to the cluster over the internet. This means that unless the cluster has terrible security, you will need a password to log in. In some cases you will be prompted for a password. In some cases a two-factor authentication will be required where you enter a one-time password or accept a request sent to your phone or email.
```

Comment:
Is the MFA sentience needed? I assumed something similar to CC would eventually be setup for the new cluster. @andy-amii

Reply:
Yup, unfortunately :( We'll be hooking into DRAC's infrastructure---specifically their OpenLDAP which is how accounts are managed---which will mean we need to use their full security stack as well.

Reply:
If I understand correctly this only works as long as you do not change your network (and maybe do not shut down laptop?) So maybe I should pitch this as "if you are going to work for a few hours and do not want to keep using multi-factor auth over and over again use this but with `ControlPersists 180m` ?

Reply:
Yup that's exactly it. This is known generally as "multiplexing" where the tcp connection is being cached and reused across ssh sessions. As long as that tcp connection stays alive, you won't need to reauthenticate. Anything that breaks the tcp connection (changing networks, putting machine to sleep, etc.) would require reestablishing the connection.

I think we should recommend the softer `ControlPersist yes`, since there really isn't any strong reason to kill the connection at a specified time limit.